app/src/lib/ui/BeneficiaryList/ListWithOrientation.svelte

```diff
@@ -29,7 +29,13 @@
 		openComponent.open({
 			component: ChangeOrientationForm,
 			props: {
-				notebooks: [{ id: beneficiary.notebook.id, beneficiaryId: beneficiary.id }],
+				notebooks: [
+					{
+						id: beneficiary.notebook.id,
+						beneficiaryId: beneficiary.id,
+						beneficiaryStructures: beneficiary.structures,
```

Comment:
```suggestion
						hasOrientation: beneficiary.structures.length > 0,
```
ca me parait plus clair sur l'intention

Reply:
Ça déplace la règle de gestion dans les appelants, c'est pour cette raison que j'étais pas fan.